src/guide/typescript-support.md

```diff
@@ -232,6 +232,30 @@ const Component = defineComponent({
 })
 ```
 
+## Annotating emits
+
+We can annotate a payload for the emitted event. Also, all non-declared emitted events will throw a type error when called:
+
+```ts
+const Component = defineComponent({
+  emits: {
+    addBook: (payload: { bookName: string }) => {
```

Comment:
**Nitpick**: Is there any reason to use an arrow function rather than a regular function in this example?

```suggestion
    addBook(payload: { bookName: string }) {
```